src/recipes/link.ts

```diff
@@ -0,0 +1,33 @@
+import { defineRecipe } from '@pandacss/dev'
+
+const linkRecipe = defineRecipe({
+  className: 'link',
+  description: '<a> styles',
+  base: {},
+  variants: {
+    in: {
+      modal: {
+        color: 'lightblue',
+      },
+      breadcrumbs: {
+        color: 'inherit',
+        textDecoration: 'none',
+      },
+      thoughtAnnotation: {
```

Comment:
The only links that appear in the `ThoughtAnnotation` component are the [UrlIconLink](https://github.com/cybersemics/em/blob/587e1d309d9570e5ad06e9bec8eb0570867aa7d4/src/components/ThoughtAnnotation.tsx#L46) and the [EmailIconLink](https://github.com/cybersemics/em/blob/587e1d309d9570e5ad06e9bec8eb0570867aa7d4/src/components/ThoughtAnnotation.tsx#L75), so let's move this style into the `ThoughtAnnotation.tsx` file, preferably as a static object rather than a recipe.